### DISCUSSION TITLE
Icons preview in PR before merging - how to?

### DISCUSSION THREAD

**Discussion by zv09** [simple-icons/simple-icons#6507] [General]
I just made my first PR for this project. 
I also found that some contributors add a preview of the created icon to their PR, even if the branch has not been merged yet.
But I have not found a description of how to do this.
Any suggestions? thx...

**Comment by jorgeamadosoria** [comment]
Hi @zv09 !
you can use this page: https://petershaggynoble.github.io/SI-Sandbox/preview/ to generate the icon preview.

Simply upload your SVG, introduce the hex for the color you have chosen, download the Icon preview and then copy it to your PR.

**Reply by zv09** [reply]
thx, 
did it, but Safari on macOS X is saving only transparent canvas, trying other browser was successful with full preview...

**Reply by jorgeamadosoria** [reply]
> thx,
> did it, but Safari on macOS X is saving only transparent canvas, trying other browser was successful with full preview...

sorry to hear that, I usually work on Chrome so I wasn't aware of that problem. I hope you made it work with a different browser in the end!

**Reply by zv09** [reply]
yep, I did.. 
Would you check something else in this case: 
Look in my PR - the style of the saved preview based on URL you had provided: 
https://github.com/simple-icons/simple-icons/pull/6502

and another one style of preview in the another PR, it different.. 
https://github.com/simple-icons/simple-icons/pull/6477

so, Q is there exist another preview script online?

**Reply by jorgeamadosoria** [reply]
> #6477

There are more than one Preview generator, made by maintainers of this project. Any will do, none is more authoritative than any other.

In the examples you provided, notice that the difference is that you downloaded the SVG preview, and @dirien simply took a screen shot of the page (which also included the parameter inputs and such).
Either is fine. The goal is to show the preview so we can review how the icon is supposed to look like, as well as the color it's supposed to have.

**Comment by adamrusted** [comment]
To clarify, the most recently updated version of our preview generator is at https://petershaggynoble.github.io/SI-Sandbox/preview/.
Please use that one, as opposed to the MDI one, as I _believe_ ours was more recently updated by @PeterShaggyNoble.
